tests/test_views.py

```diff
@@ -1,15 +1,22 @@
 import pytest
+from urllib.parse import urlencode
+import usaddress
 
 
 def test_api_parse_succeds(client):
-    # TODO: Finish this test. Send a request to the API and confirm that the
-    # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+    test_address_components, test_address_type = usaddress.tag(address_string)
+    get_param = urlencode({'address': address_string}) 
 
+    response = client.get(f'/api/parse/?{get_param}')
+    assert response.status_code == 200
+    assert response.data['input_string'] == '123 main st chicago il' 
+    assert response.data['address_components'] == test_address_components
+    assert response.data['address_type'] == test_address_type
 
 def test_api_parse_raises_error(client):
-    # TODO: Finish this test. The address_string below will raise a
-    # RepeatedLabelError, so ParseAddress.parse() will not be able to parse it.
     address_string = '123 main st chicago il 123 main st'
-    pytest.fail()
+    get_param = urlencode({'address': address_string}) 
+    response = client.get(f'/api/parse/?{get_param}')
```

Comment:
Ditto passing args directly to `client.get()`.

Reply:
Per the comment above, based on the resources you shared I used the test client functionality to pass in the address string as the query parameter